samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -506,7 +515,11 @@ object SamzaContainer extends Logging {
               case _ => null
             }
 
-            val storeDir = if (changeLogSystemStreamPartition != null) {
+            // We want to use the job store path from the config for standalone deployment model for both changelog
+            // enabled & disabled stores. We preserve the existing behaviour in yarn where non-changelog stores
+            // use "user.dir" for store base directory even if the environment variable LOGGED_STORE_BASE_DIR is set.
+            // TODO: Identify the impact of using logged store directories for non-changelog stores.
```

Comment:
> Identify the impact of using logged store directories for non-changelog stores.

Existing behavior for non changelogged stores is that whenever the samza container restarts, we will begin with the clean slate. That will be violated if non-changelog stores are put in change-log directory and SamzaContainer starts again in the same host after a restart. 

Typical scenario I've seen is to use rocksDb as a cache in front of remote store.